test/spec/SearchApi/Support/GeoCoderURLBuilderSpec.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace spec\SearchApi\Support;
+
+use SearchApi;
+use SearchApi\Models as Models;
+use SearchApi\Providers as Providers;
+use SearchApi\Support as Support;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+/**
+ * GeoCoderURLBuilderSpec - Spec integration test for the GeoCoderURLBuilder (higher level functions)
+ */
+class GeoCoderURLBuilderSpec extends ObjectBehavior {
+  function it_is_initializable() {
+    $this->beConstructedWith( new Models\GeoCoordinate( 40.714224, -73.961452 ) );
```

Comment:
you should factor out this test coordinate into a class variable and just reference it by name so we know if it is a valid value or an invalid value that your testing. It mainly helps readability